thelastinuit/21-aug-triple_trouble.exs

```diff
@@ -0,0 +1,18 @@
+# https://www.codewars.com/kata/5704aea738428f4d30000914
+
+defmodule Codewars.WeirdString do
+  def triple_trouble(one, two, three) do
+    # For fancy chars, use graphemes. it's slow, though.
+    chars_one = one |> String.codepoints
```

Comment:
piping biatch is translated from cejudaismo to english as
The pipe operator which looks like |> takes the result of one expression, and passes it on.

check it out here: https://elixirschool.com/en/lessons/basics/pipe-operator/